.teamcity/_self/projects/WebApp.kt

```diff
@@ -18,6 +18,7 @@ object WebApp : Project({
 
 	buildType(RunAllUnitTests)
 	buildType(CheckCodeStyleBranch)
+	buildType(DataViewsChangelogCheck)
```

Comment:
Historically, we've tried to keep the number of checks minimal, IIRC, due to lack of TC agents which was causing congestion. While I'm not 100% sure that's the case today, I wonder if it might make sense to make this check part of another check, like the Unit Tests one, that already has a bunch of additional checks, like type checks and storybook smoke tests.

Reply:
That's good context, and this was something I was unsure of. I'm happy to fold it into `RunAllUnitTests` or elsewhere.

Reply:
I'd be fine to make it part of the "Unit Tests" build for now, yes. We can always move it out if we want.